src/axom/inlet/SphinxWriter.hpp

```diff
@@ -39,14 +39,30 @@ namespace inlet
 class SphinxWriter : public Writer
 {
 public:
+  /*!
+  *******************************************************************************
+  * \brief Supported phinx style options
+  *******************************************************************************
+  */
+  enum class Style
+  {
+    // FIXME: These names are not clear at all
+    Table,  // Tables for the child Fields/Functions of each Container

```

Comment:
I think nested is clear (at least how I see it) maybe focus on figuring out a better name for the overloaded "Table".  Singular or Individuals? or possibly just non-nested

Reply:
I think it's within the realm of possibility that we'd want a third style of Sphinx documentation - the other consideration is readability in the constructor call:
```cpp
SphinxWriter("docs.rst", "My Docs", true);
// vs...
SphinxWriter("docs.rst", "My Docs", SphinxWriter::Style::Nested);
```